flatpak/manifest_tool/flutter_ops.py

```
@@ -0,0 +1,416 @@
+            extra_sources.append(candidate.name)
+        helper = output_path / "setup-flutter.sh"
+        if helper.exists():
+            extra_sources.append({"type": "file", "path": helper.name})
+        module["sources"] = [
+            {
+                "type": "archive",
+                "path": archive_name,
+                "sha256": sha256,
+                "strip-components": 1,
+            }
+        ] + extra_sources
```

Ruff (0.13.1)

Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)